File: baekjoon/js/5597.js
const fs = require("fs");
const buf = fs.readFileSync(0, "utf-8").toString().trim();

const nums = buf.split("\n").map(Number);

const result = [];

for (let i = 1; i <= 30; i++) {
  if (!nums.includes(i)) {
    result.push(i);
  }
}

console.log(result.slice(0, 2).join("\n"));

File: baekjoon/rust/5597.rs
use std::io::{read_to_string, stdin};

fn main() {
    let buf = read_to_string(stdin()).unwrap();
    let inputs: Vec<i32> = buf
        .split_whitespace()
        .map(|v| v.parse::<i32>().unwrap())
        .collect();

    let mut result = Vec::new();

    for n in 1..=30 {
        if !inputs.contains(&n) {
            result.push(n.to_string());
        }

        if result.len() == 2 {
            break;
        }
    }

    print!("{}", result.join("\n"));
}
